spec/features/user_visits_form_spec.rb

```diff
@@ -48,15 +49,17 @@
   end
 
   it 'should show confirmation page after a succesful form submit' do
-    stub_request(:post, ZENDESK_TICKETS_URL).to_return(:status => 201, :body => {"ticket":{"id": 123, "subject":"Test","comment":{"value":"some comment"}}}.to_json, :headers => {})
+    ticket_number = 123456
+    stub_request(:post, ZENDESK_TICKETS_URL).to_return(:status => 201, :body => {"ticket":{"id":ticket_number}}.to_json, :headers => { "Content-Type": "application/json" })
+    stub_request(:post, ZENDESK_UPLOADS_URL).to_return(:status => 201, :body => {"upload":{"token":ticket_number}}.to_json, :headers => { "Content-Type": "text/plain" })
+    stub_request(:put, "#{ZENDESK_TICKETS_URL}/#{ticket_number}").to_return(:status => 200, :body => {"ticket":{"id":ticket_number}}.to_json, :headers => { "Content-Type": "application/json" })
     submit_valid_form
-    expect(page).to have_content('Your ticket has been created with the id #123')
-  end
+    expect(page).to have_content('Your ticket has been created with the id #123456')
 
-  it 'should show confirmation page after a succesful form submit even if there\'s no ticket id' do
-    stub_request(:post, ZENDESK_TICKETS_URL).to_return(:status => 201, :body => {"ticket":{"subject":"Test","comment":{"value":"some comment"}}}.to_json, :headers => {})
-    submit_valid_form
-    expect(page).to have_content('Your ticket has been created with the id #unknown')
+    File.delete('./tmp/something-something.yml')
+    File.delete('./tmp/something-something-msa.yml')
+    File.delete('./tmp/experian-something-something.yml')
+    File.delete('./tmp/post-office-something-something.yml')
```

Comment:
Use stub IDP names.

Reply:
@chrisholmes has said this is acceptable, the initial decision to not include any references to idp's referred to not sharing any styling or content that they owned.